pkg/koordlet/util/container.go

```diff
@@ -203,6 +211,14 @@ func GetContainerCurTasks(podParentDir string, c *corev1.ContainerStatus) ([]int
 	return system.GetCgroupCurTasks(cgroupPath)
 }
 
+func GetContainerCurNumaStat(podParentDir string, c *corev1.ContainerStatus) ([]system.NumaMemoryPages, error) {
```

Comment:
To support different cgroup versions, it is recommended to add a `(*resourceexecutor.CgroupReader).ReadNumaStat()` instead of the `GetContainerCurNumaStat()`.